### DISCUSSION TITLE
HUD Depth Configuration?

### DISCUSSION THREAD

**Discussion by LoquaciousTenacious** [LivingFray/HaloCEVR#74] [General]
I'll preface this by saying that this mod is amazing, unbelievably immersive, and probably my favorite VR experience of all time. It's really fantastic. The one thing that keeps if from being _perfectly_ immersive for me is the depth at which the HUD is drawn. Seeing elements like the shield, motion tracker, ammo, etc floating off in the distance, frequently phased through geometry, when diagetically they exist on the inside of a helmet visor takes me out of the experience slightly.

Is there any way to bring those HUD elements closer to the player viewport _without_ adjusting the draw distance of the weapon reticle? If they could be decoupled this mod would go from a 9.5/10 to a 12/10.

**Comment by RocketMan2007** [comment]
This is by far my favorite VR mod, but I would agree that the HUD does take away a small bit from the emersion. Even if there were a way to toggle the HUD display on/off would be fantastic.

**Comment by LivingFray** [comment]
Try the latest release. I've split the UI distance/scaling setting into 3 parts (UI, menu, crosshair). 
Pro tip: If the UI distance is negative it will always be behind you and effectively disabled.
